pkg/server/podsubresourceproxy_handler_test.go

```diff
@@ -163,7 +174,23 @@ func TestPodSubresourceProxyingHandler(t *testing.T) {
 				return
 			}
 			require.Equal(t, http.StatusOK, result.StatusCode, "Unexpected status code: %s", string(bytes))
-			require.Equal(t, tc.expectedPath, r.URL.Path, "Unexpected path")
+			if tc.expectedPath != "" {
```

Comment:
How is `expectedPath` useful now ? Why not remove it ?

Reply:
replaced by the expectedproxiedPath

Reply:
done